mongo/src/main/java/com/redhat/lightblue/mongo/crud/BasicDocSaver.java

```diff
@@ -53,7 +53,7 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(BasicDocSaver.class);
 
-    private int batchSize = 64;
+    private final int batchSize = 64;
```

Comment:
Probably the datasources.json, but I would be fine if we spawned a ticket to address this later as a configuration.